Add the -n for pod in kubectl command (#16798)
kubernetes/website: content/en/docs/contribute/style/style-guide.md

1. Display information about a Pod:

        kubectl describe pod <pod-name> -n <namespace>

>     where `<pod-name>` is the name of one of your Pods, `<namespace>` is the Pod's namespace.

Reviewer: ```suggestion
```

>     If the namespace of the pod is default, can be without '-n <namespace>'.

Reviewer: ```suggestion
    If the namespace of the pod is `default`, you can omit the '-n' parameter.
```
